app/src/main/java/de/tum/in/tumcampusapp/component/ui/openinghour/model/Location.kt

```diff
@@ -25,8 +26,11 @@ data class Location(@PrimaryKey
                     var name: String = "",
                     var address: String = "",
                     var room: String = "",
+                    @SerializedName("transport_station")
                     var transport: String = "",
+                    @SerializedName("opening_hours")
                     var hours: String = "",
+                    @SerializedName("infos")
```

Comment:
I guess you did not change anything with any database model except this, so a migration is not needed right?

Reply:
Yes, no migration is needed